monitoring/loadtest/requirements.txt

```diff
@@ -0,0 +1,7 @@
+google-auth==1.6.3
```

Comment:
The auth lib requirements are in monitorlib/requirements.txt included below; they don't need to be additionally included here (and we don't want to because that increases the likelihood of version mismatch).  I think we only need to specify `locust` here.